src/cfdcore_schnorrsig.cpp

```diff
@@ -312,6 +314,54 @@ Pubkey SchnorrUtil::ComputeSigPoint(
   return ConvertSecpPubkey(secp_sigpoint);
 }
 
+Pubkey SchnorrUtil::ComputeSigPointBatch(
+    const std::vector<ByteData256> &msg,
+    const std::vector<SchnorrPubkey> &nonces, const SchnorrPubkey &pubkey) {
+  auto ctx = wally_get_secp_context();
+
+  if (msg.size() != nonces.size() || msg.size() < 1) {
+    throw CfdException(
+        CfdError::kCfdIllegalArgumentError,
+        "Expected same number of messages or nonces, and at least one "
+        "message.");
+  }
+
+  std::vector<secp256k1_xonly_pubkey> secp_nonces;
```

Comment:
unused value?

Reply:
Yes sorry fixed. (Also changed the type of `i` in the for loop from `int` to `size_t` as it was generating warnings.

Reply:
>Also changed the type of i in the for loop from int to size_t as it was generating warnings.

Ok!

Reply:
It looks like you forgot to turn it off. 😃